addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.plot_curve",
+    "name": "Plot Curve",
+    "description": "A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
+    "previews": [],
+    "elements": [
+        {
+            "name": "Plot Curve",
+            "identifier": "com.seeq.addons.plot_curve.datalabfunctions",
+            "known_aliases": ["PlotCurve", "Plot Curve"],
+            "description":"A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+            "type": "AddOnTool",
+            "path": "data-lab-functions",
+            "notebook_file_path": "PlotCurve.ipynb",
+            "configuration_schema": {
+                "type": "object",
+                "properties": {
+                    "display": {
+                        "type": "object",
+                        "properties": {
+                            "icon": {
+                                "type": "string",
+                                "default": "fa fa-line-chart"
```

Comment:
[nit] I noticed the Add-on icon above on line 7 is set to `fa-edit`. It might make more sense to have the Add-on's icon that displays in the Add-on Manager be the same icon that the user sees in the Tools panel.

Reply:
In the next push!